### DISCUSSION TITLE
Packaging 6.2.54

### DISCUSSION THREAD

**Discussion by ikedas** [sympa-community/sympa#1612] [Team Posts]
* `Unicode::UTF8` is required at build time, and recommended at run time.
* `t/Message_urlize.t` fails with earlier version (before 5.500) of MIME-tools.  For example RHEL/CentOS 6 bundles perl-MIME-tools-4.427.

**Comment by racke** [comment]
Added prerequisites for `Unicode::UTF8` to Debian packaging: https://salsa.debian.org/sympa-team/sympa/-/commit/7e4fcb0a8f7087f5f8616827023f706a2d30b9e3.

Should I add a run time dependency on the MIME tools version in addition to the build time one?

**Comment by ikedas** [comment]
I forgot to write.

> t/Message_urlize.t fails with earlier version (before 5.500) of MIME-tools. For example RHEL/CentOS 6 bundles perl-MIME-tools-4.427.

In case of earlier version of MIME-tools, apply sympa-community/sympa#890 to solve problem.
